spec/unit/module/forge_spec.rb

```diff
@@ -23,6 +23,12 @@
     end
   end
 
+  describe "implementing the generic hash spec" do
+    it "should implement {type: forge}" do
```

Comment:
Should this test include all the generic hash keys?

Reply:
Might as well. Adding them.

Reply:
Not sure if you wanna update this description also